app/routes/irrigation-systems.js

```diff
@@ -16,37 +18,12 @@ function createModel(errorMessage, errorSummary, currentData, plannedData) {
                 }
             },
             items: [
-
-                {
-                    value: "Trickle",
-                    text: "Trickle",
-                    checked: isChecked(currentData,'Trickle')
-                },
-                {
-                    value: "Boom irrigator",
-                    text: "Boom irrigator",
-                    checked: isChecked(currentData,'Boom irrigator')
-                },
-                {
-                    value: "Ebb and flood or capillary bed",
-                    text: "Ebb and flood or capillary bed",
-                    checked: isChecked(currentData,'Ebb and flood or capillary bed')
-                },
-                {
-                    value: "Sprinklers or mist",
-                    text: "Sprinklers or mist",
-                    checked: isChecked(currentData,'Sprinklers or mist')
-                },
-                {
-                    value: "Rain gun",
-                    text: "Rain gun",
-                    checked: isChecked(currentData,'Rain gun')
-                },
-                {
-                    value: "Not currently irrigating",
-                    text: "Not currently irrigating",
-                    checked: isChecked(currentData,'Not currently irrigating')
-                }
+                setLabelData(currentData, 'Trickle'),
```

Comment:
why not change setLabelData in such a way that it accepts as parameters: currentData & an array of values:
const setLabelData = (currentData, arrVals) => {
	...
	// maps through arrVals & creates an array of items ... 
}
instead of calling setLabelData multiple times --> more efficient than calling the same function multiple times

Reply:
also, will the value & text of a checkbox, radiobox always be the same?
Just asking, because if at all the content changes and they become different, it may mean we break this function again

Reply:
good spot i will update the code